components/constellation/constellation.rs

```diff
@@ -1456,11 +1468,11 @@ where
             FromCompositorMsg::Exit => {
                 self.handle_exit();
             },
-            FromCompositorMsg::GetBrowsingContext(pipeline_id, resp_chan) => {
-                self.handle_get_browsing_context(pipeline_id, resp_chan);
+            FromCompositorMsg::GetBrowsingContext(pipeline_id, browsing_context_ipc_sender) => {
```

Comment:
`response_sender` is a actually a pretty good name here, because it describes what the channel does -- handle the response to this query.